src/Validation/OrderPropertyValidator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Validation;
+
+/**
+ * @method canHaveMultiCollo(?bool $value): bool
+ * @method canHaveSignature(?bool $value): bool
+ * @method canHaveDate(?string $value): bool
+ */
+trait OrderPropertyValidator
+{
+    public function __call(string $methodName, $args): bool
+    {
+        $schema = $this->getValidationSchema();
+        $keys   = self::METHOD_TO_KEYS[$methodName];
+        $result = $this->walkArrayByKeys($schema, $keys);
+
+        if (! $result) {
+            return false;
+        }
+
+        if (array_key_exists('enum', $result)) {
```

Comment:
Zoals besproken, is het wellicht mogelijk om isset te gebruiken op veel plekken ipv array_key_exists?